docs/data/date-pickers/base-concepts/ReferenceDateUsingValue.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import dayjs from 'dayjs';
+import { DemoContainer } from '@mui/x-date-pickers/internals/demo';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import { DatePicker } from '@mui/x-date-pickers/DatePicker';
+
+export default function ReferenceDateUsingValue() {
+  return (
+    <LocalizationProvider dateAdapter={AdapterDayjs}>
+      <DemoContainer components={['DatePicker']}>
+        <DatePicker
+          defaultValue={dayjs('2022-04-17')}
+          views={['year', 'month', 'day']}
```

Comment:
Nitpick: We are not mentioning the month part for this demo, hence, having all views could be omitted. 🤔

Reply:
My issue is that we a demonstrating exactly the same thing as in `date-calendar`, so using a different set of props feels wrong to me.
I would either add the views on both or on neither.

Reply:
Sure thing, consistency sounds good as well. 👍 
Was just thinking from the perspective that there is extra code/props that are not necessarily worth it. 🙈